filebeat/fileset/modules.go

```diff
@@ -240,3 +241,29 @@ func (reg *ModuleRegistry) GetProspectorConfigs() ([]*common.Config, error) {
 	}
 	return result, nil
 }
+
+func (reg *ModuleRegistry) Setup(esClient *elasticsearch.Client) error {
```

Comment:
do we need the full `elasticsearch.Client` here? If we define an interface with methods required, it get's us an idea, well, what's really required for ModuleRegistry + we can easily mock the client in tests.